pkg/ansible/flags/flag.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // AnsibleOperatorFlags - Options to be used by an ansible operator
 type AnsibleOperatorFlags struct {
-	watch.WatchFlags
+	watch.Flags
```

Comment:
Even though `watch.WatchFlags` is an internal type, being embedded here means that changing it to `watch.Flags` is a breaking change to `AnsibleOperatorFlags`, which is exported. So this is a breaking change that should be reverted if we decide to avoid breaking changes.